packages/paste-core/components/base-radio-checkbox/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@twilio-paste/base-radio-checkbox",
+  "version": "1.0.0",
+  "category": "user input",
+  "status": "beta",
+  "description": "A shared base component for radio and checks",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn compile",
+    "build:dev": "yarn clean && yarn compile:dev",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.build.tsbuildinfo && rm -rf .rpt2_cache",
+    "compile": "rollup -c --environment NODE_ENV:production",
+    "compile:dev": "rollup -c --environment NODE_ENV:development",
+    "prepublishOnly": "yarn build",
+    "type-check": "tsc --noEmit"
+  },
+  "dependencies": {
+    "react-autosize-textarea": "7.0.0"
```

Comment:
Do we need `react-autosize-textarea` here?

Reply:
Nope, good catch